### PR TITLE
[prefers-color-scheme article] Add info about DevTools debugging and Puppeteer testing

### PR DESCRIPTION
**Pull request by tomayac**
Fixes https://github.com/GoogleChrome/web.dev/issues/1886.

Changes proposed in this pull request:

- Add info about debugging dark mode with Chrome DevTools
- Add info about testing dark mode with Puppeteer

### CODE REVIEW

File: src/site/content/en/blog/prefers-color-scheme/index.md [hunk 1]
```diff
@@ -464,6 +464,34 @@ in order to see the theme color and favicon changes, open the
   });
 ```
 
+## Debugging and testing dark mode
+
+### Emulating `prefers-color-scheme` in DevTools
+
+Switching the entire operating system's color scheme can get annoying real quick,
+so in Chrome DevTools, we now allow you to emulate the user's preferred color scheme
+in a way that only the currently visible tab is affected.
```
Comment (mathiasbynens) [inline src/site/content/en/blog/prefers-color-scheme/index.md]: Slightly simpler:

```suggestion
so Chrome DevTools now allows you to emulate the user's preferred color scheme
in a way that only affects the currently visible tab.
```

File: src/site/content/en/blog/prefers-color-scheme/index.md [hunk 2]
```diff
@@ -464,6 +464,34 @@ in order to see the theme color and favicon changes, open the
...
+You can find the emulation setting in the Rendering section of the Chrome DevTools.
```
Comment (mathiasbynens) [inline src/site/content/en/blog/prefers-color-scheme/index.md]:
```suggestion
You can find the emulation setting in the Rendering drawer of the Chrome DevTools.
```

File: src/site/content/en/blog/prefers-color-scheme/index.md (hunk 1 above)
Comment (mathiasbynens) [reply src/site/content/en/blog/prefers-color-scheme/index.md]: maybe make "now allows you" a link to https://developers.google.com/web/updates/2019/10/devtools#userpreferences

**Comment by mathiasbynens** [comment]
This is great! Thank you, Thomas :+1: 

I left some minor suggestions.

File: src/site/content/en/blog/prefers-color-scheme/index.md [hunk 3]
```diff
@@ -464,6 +464,35 @@ in order to see the theme color and favicon changes, open the
   });
 ```
 
+## Debugging and testing dark mode
+
+### Emulating `prefers-color-scheme` in DevTools
+
+Switching the entire operating system's color scheme can get annoying real quick,
+so in Chrome DevTools, we now allow you to emulate the user's preferred color scheme
```
Comment (mathiasbynens) [inline src/site/content/en/blog/prefers-color-scheme/index.md]: This line should be removed now (sorry, GitHub wouldn't let me suggest a multi-line patch before)

Comment (tomayac) [reply src/site/content/en/blog/prefers-color-scheme/index.md]: Ooops, fixed. Thanks!

File: src/site/content/en/blog/prefers-color-scheme/index.md [hunk 4]
```diff
@@ -464,6 +464,34 @@ in order to see the theme color and favicon changes, open the
...
+### Emulating `prefers-color-scheme` in DevTools
+
+Switching the entire operating system's color scheme can get annoying real quick,
+so Chrome DevTools now allows you to emulate the user's preferred color scheme
+in a way that only affects the currently visible tab.
+You can find the emulation setting in the Rendering drawer of the Chrome DevTools.
+
+<figure class="w-figure">
+  <img src="devtools-emulate.png" alt="Emulate the CSS media feature prefers-color-scheme" width="945" height="652">
+  <figcaption class="w-figcaption">Emulate the CSS media feature <code>prefers-color-scheme</code> in Chrome DevTools.</figcaption>
```
Comment (kaycebasques) [inline src/site/content/en/blog/prefers-color-scheme/index.md]:
```suggestion
```
Caption isn't needed here

File: src/site/content/en/blog/prefers-color-scheme/index.md [hunk 5]
```diff
@@ -464,6 +464,34 @@ in order to see the theme color and favicon changes, open the
...
+
+### Emulating `prefers-color-scheme` in DevTools
+
+Switching the entire operating system's color scheme can get annoying real quick,
+so Chrome DevTools now allows you to emulate the user's preferred color scheme
+in a way that only affects the currently visible tab.
+You can find the emulation setting in the Rendering drawer of the Chrome DevTools.
+
+<figure class="w-figure">
+  <img src="devtools-emulate.png" alt="Emulate the CSS media feature prefers-color-scheme" width="945" height="652">
```
Comment (kaycebasques) [inline src/site/content/en/blog/prefers-color-scheme/index.md]:
```suggestion
  <img src="devtools-emulate.png" alt="A screenshot of the 'Emulate CSS media feature prefers-color-scheme' option that is located in the Rendering tab of Chrome DevTools" width="945" height="652">
```

File: src/site/content/en/blog/prefers-color-scheme/index.md [hunk 6]
```diff
@@ -464,6 +464,34 @@ in order to see the theme color and favicon changes, open the
...
 ```
 
+## Debugging and testing dark mode
+
+### Emulating `prefers-color-scheme` in DevTools
+
+Switching the entire operating system's color scheme can get annoying real quick,
+so Chrome DevTools now allows you to emulate the user's preferred color scheme
+in a way that only affects the currently visible tab.
+You can find the emulation setting in the Rendering drawer of the Chrome DevTools.
```
Comment (kaycebasques) [inline src/site/content/en/blog/prefers-color-scheme/index.md]:
```suggestion
Open the [Command Menu](https://developers.google.com/web/tools/chrome-devtools/command-menu), start typing `Rendering`, run the `Show Rendering` command, and then change the **Emulate CSS media feature prefers-color-scheme** option.
```

File: src/site/content/en/blog/prefers-color-scheme/index.md [hunk 7]
```diff
@@ -4,7 +4,7 @@ subhead: Overhyped or necessity? Learn everything about dark mode and how to sup
 authors:
   - thomassteiner
 date: 2019-06-27
-updated: 2019-06-28
+updated: 2019-11-18
```
Comment (kaycebasques) [inline src/site/content/en/blog/prefers-color-scheme/index.md]:
```suggestion
updated: 2019-11-25
```

**Pull request merged**